src/style.css

```diff
@@ -0,0 +1,236 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #f5f5f5;
+  flex-direction: column;
+  display: flex;
```

Comment:
To follow the CSS best practices, we should consider starting by `display: flex` then `flex-direction: column`